### ISSUE TITLE
[Q] How rename downloaded torrent file

### ISSUE THREAD

**Issue by nvcken** [fagga/transmission-remote-cli#169]
The case is 
- Install transmission & transmission-daemon on CentOS
- Enable Web UI for remoting
- Add torrent file from Web UI remote
- Get Error: Unable to save resume file: File name too long

So far I can't find the way to rename downloaded file from Web UI.
Does transmission-remote-cli could help ?

**Comment by fagga** [comment]
Download the file (right click, "save as...") and rename the torrent in your
download folder. Then you should be able to add it in the Web UI or any other
Transmission client.

**Comment by nvcken** [comment]
Nope, Error "File name too long" is the file in torrent contents, not the name of .torrent file

**Comment by fagga** [comment]
There is an option to rename files in transmission-remote-cli ('F' in Files
view), but I've never used it and don't know how it works exactly and how
buggy it is.

If transmission refuses to add the torrent, you're out of luck and have to
edit the torrent file somehow (not with trcli) before adding it.
